apps/web/src/components/Footer/VersionLink.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { useIntl } from "react-intl";
+
+import { Link } from "@gc-digital-talent/ui";
+import { commonMessages } from "@gc-digital-talent/i18n";
+
+const VersionLink = () => {
+  const intl = useIntl();
+  let content = process.env.COMMIT_HASH;
+  if (process.env.VERSION) {
+    content = process.env.VERSION;
+  }
+
+  if (!content) {
+    return null;
+  }
+
+  return (
+    <span>
+      {" "}
+      &mdash;{" "}
+      {intl.formatMessage({
+        id: "C3fUwm",
+        defaultMessage: "Version",
```

Comment:
The French translation for **Version** is the same as in English, **Version**.

Reply:
Awesome, thanks! Added in [add translation](https://github.com/GCTC-NTGC/gc-digital-talent/pull/8354/commits/206e123a321baf7388e1e8fa5107dd6ed98e8e8b)